### ISSUE TITLE
Giveaways ending in future day(s) show ending "today" 

### ISSUE THREAD

**Issue by CynnaMonkey** [jagrosh/GiveawayBot#28]
I posted a giveaway scheduled to end in 2 days. In the post it says "ends at: Today" with the time it posted which was the past. It does correctly say "Time remaining: 2 days".

**Comment by jagrosh** [comment]
Do you encounter the same issue across multiple clients (desktop, browser, and mobile)?
What do other members of the server see?

**Comment by Purpzie** [comment]
That one seems like a Discord embed issue, not with GiveawayBot. Basically, the embed only gets a timestamp (to allow it to be displayed in any timezone), and it's up to Discord to decide how it gets displayed. GiveawayBot doesn't control that part of the embed.

There is already a report on this which you can find [here](https://trello.com/c/RO4zrt25/) and you can also add your own canrepro to it [here](https://discord.gg/discord-testers) with `!canrepro 1626 | <your device information>` in the `#android-bug-reports` channel.

**Comment by Sanduhr32** [comment]
We can controll the universal time but we can not change the client rendering and since desktop and android are hit by this issue I assume it's a general converting issue in any discord client.

I personally also encountered this issue and tried to resolve it but quickly noticed it client sided and looked the code up.
The code never modifies the ending time, it just sets it on the creation of a giveaway.
### For Developers
Yes we do resend the embed timestamp each time we update the embed for avoiding its removal.

**Comment by jagrosh** [comment]
Yeah, this is a bug with Discord's rendering of the timestamp, and is out of our power to fix because it only happens on some clients (we can't use an intentionally-wrong timestamp to get around this, because then the timestamp would be wrong for other people)
